sdk/containerregistry/Azure.Containers.ContainerRegistry/README.md

```diff
@@ -35,15 +35,37 @@ az acr create --name MyContainerRegistry --resource-group MyResourceGroup --loca
 
 ### Authenticate the client
 
-The [Azure Identity library][identity] provides easy Azure Active Directory support for authentication.
+The [Azure Identity library][identity] provides easy Azure Active Directory support for authentication.  The `DefaultAzureCredential` assumes the `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, and `AZURE_CLIENT_SECRET` environment variables are set.For more information refer to [Azure Identity environment variables](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/identity/Azure.Identity#environment-variables).
```

Comment:
It may not be accurate to say `DefaultAzureCredential` assumes these env vars, as only the `EnvironmentalCredential` uses them?

nit: missing space after "variables are set."

Reply:
Yes, good call.  I fixed that in my last push.